Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/ErrorTests.cs

```diff
@@ -172,7 +162,8 @@ public void DownloadFiles_TestCancellationToken()
                 Operation = SourceOperation.Nothing,
             };
 
-            var ex = Assert.ThrowsAsync<Exception>(async () => await SFTP.DownloadFiles(source, _destination, connection, _options, _info, new CancellationTokenSource(2000).Token));
+            using var cancellationTokenSource = new CancellationTokenSource(2000);
+            var ex = Assert.ThrowsAsync<Exception>(async () => await SFTP.DownloadFiles(source, _destination, connection, _options, _info, cancellationTokenSource.Token));
             Assert.IsTrue(ex.Message.Contains("No files transferred."));
             Assert.IsTrue(ex.Message.Contains("Error: The operation was canceled.."));
             Assert.IsTrue(Helpers.SourceFileExists($"{source.Directory}/{source.FileName}"));

```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/ff45de45718f2d57ed3ea332c72aa78c993eed9f/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/ErrorTests.cs#L197
No need to write to console.